docker/README.md

```diff
@@ -3,141 +3,137 @@
 [![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
 [![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
 
-An Internet of Things Agent for a JSON based protocol (with
```

Comment:
Many of the changes in this .md seems to be of prettier nature... which wonders me a bit given that as far as I understand we already did the prettifi-cation in a past PR.

I feel curiosity. Why we are seeing again this kind of changes?

Reply:
The existing `npm run prettier:text` task only updates documentation markdown - the command misses `Docker\README.md`

The husky hook covers every markdown file - hence the `Docker\README.md` is now properly formatted.

Reply:
Thanks for the clarification. I wonder if prettie:text should cover also Docker\README.md, as husky does, but probably it doesn't worth the time.

NTC